docs/source/load-balancing/default-policy.md

```diff
@@ -35,36 +33,34 @@ let default_policy = DefaultPolicy::builder()
 
 ### Semantics of `DefaultPolicy`
 
-#### Preferred Datacenter
+#### Preferences
+
+The `preferences` field in `DefaultPolicy` allows the load balancing
+policy to prioritize nodes based on their location. When `preferences` are set
+to `ReplicaLocationPreference::Datacenter(my_dc)`, the policy will treat nodes
+in `my_dc` as "local" nodes, and nodes in other datacenters as "remote" nodes.
+This affects the order in which nodes are returned by the policy when selecting
+replicas for read or write operations. If `preferences` are set to `Any`,
+the policy will treat all nodes as local nodes.
 
-The `preferred_datacenter` field in `DefaultPolicy` allows the load balancing
-policy to prioritize nodes based on their location. When a preferred datacenter
-is set, the policy will treat nodes in that datacenter as "local" nodes, and
-nodes in other datacenters as "remote" nodes. This affects the order in which
-nodes are returned by the policy when selecting replicas for read or write
-operations. If no preferred datacenter is specified, the policy will treat all
-nodes as local nodes.
+`preferences` allow the load balancing policy to prioritize nodes based on their
+availability zones in the preferred datacenter, too. When `preferences` are set
+to `DatacenterAndRack(my_dc, my_rack)`, the policy will first return replicas
+in the local rack in the preferred datacenter, and then the other replicas
+in the datacenter (followed by remote nodes).
```

Comment:
I think this paragraph could be more clear if it didn't mention implementation details so much. It should explain that the load balancing policy has three modes and they behave such and such without referring to `preferences` and `ReplicaLocationPreference`.